client/me/purchases/cancel-purchase/index.jsx

```diff
@@ -213,7 +206,7 @@ class CancelPurchase extends React.Component {
 				<CompactCard className="cancel-purchase__product-information">
 					<div className="cancel-purchase__purchase-name">{ purchaseName }</div>
 					<div className="cancel-purchase__site-title">{ siteName || siteDomain }</div>
-					<ProductLink purchase={ purchase } selectedSite={ this.props.selectedSite } />
+					<ProductLink purchase={ purchase } siteSlug={ this.props.siteSlug } />
```

Comment:
This change looks wrong -- the component still expects `selectedSite`.  So this seems to still be causing the "View Plan Features" link to disappear on the cancellation form, for all products including WordPress.com.

Reply:
Reverted in 1e6c160